service/controller/v24/version_bundle.go

```diff
@@ -22,6 +22,11 @@ func VersionBundle() versionbundle.Bundle {
 				Description: "Introduce explicit resource reservation for OS resources and container runtime.",
 				Kind:        versionbundle.KindAdded,
 			},
+			{
+				Component:   "kvm-operator",
+				Description: "Fix NTP server configuration on tenant nodes.",
```

Comment:
Is this a good changelog entry, given the recent discussion?

Reply:
It is good for me now. No objections.